nifi-nar-bundles/nifi-lumberjack-bundle/nifi-lumberjack-processors/src/main/java/org/apache/nifi/processors/lumberjack/ListenLumberjack.java

```diff
@@ -62,12 +62,13 @@
 
 import com.google.gson.Gson;
 
+@Deprecated
 @InputRequirement(InputRequirement.Requirement.INPUT_FORBIDDEN)
 @Tags({"listen", "lumberjack", "tcp", "logs"})
-@CapabilityDescription("Listens for Lumberjack messages being sent to a given port over TCP. Each message will be " +
+@CapabilityDescription("This processor is deprecated and will be removed in the near future. Listens for Lumberjack messages being sent to a given port over TCP. Each message will be " +
```

Comment:
maybe rephrase to "may be removed from the distribution". Deprecated things have a tendency to linger ;)